hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/disk/OnDiskWritableKvStateMetrics.java

```diff
@@ -60,7 +60,7 @@ public OnDiskWritableKvStateMetrics(
                 .withFormat("%,d");
         metrics.getOrCreate(totalUtilizationConfig);
 
-        final double relativeFactor = 100.0 / maxCapacity;
+        final double relativeFactor = 100.0 / Math.max(1.0, maxCapacity); // in some tests, maxCapacity is 0
```

Comment:
This is just a workaround for now. We need to fix the tests in the future.

Reply:
Makes sense